Unity3D/Assets/RosSharp/Editor/UrdfImporter/UrdfRobotExtensions.cs

```diff
@@ -59,5 +78,93 @@ public static void setKinematic(GameObject robot, bool isKinematic)
             foreach (Rigidbody rigidbody in rigidbodies)
                 rigidbody.isKinematic = isKinematic;
         }
+
+        public static bool ImportUrdfToUnity(Robot robot)
+        {
+            // Create URDF folder
+            if (!AssetDatabase.IsValidFolder("Assets/Urdf"))
```

Comment:
We should not restrict users to put their robots into Assets/Urdf.
Note that currently users can put their robot file anywhere in the asset folder. The only restriction we currently have is, that the urdf marks the root of the ROS package.

Reply:
Would it be ok to have `Assets/<robot_package>` for each robot imported?

Reply:
Or should the user be prompted where to import it?